### DISCUSSION TITLE
Responsive menu not closing when clicking on a link

### DISCUSSION THREAD

**Discussion by josemanuelsh** [laravel/nova-issues#4027] [Ideas & Feature Requests]
- Laravel Version: 9
- Nova Version: 4.2.2
- PHP Version: 8.0
- Database Driver & Version:
- Reproduction Repository: https://github.com/laravel/nova-dusk-suite

### Description:

The responsive menu is used when the window is less than 1024px wide. After opening the menu and clicking on a link, the menu remains visible until manually closed.

### Detailed steps to reproduce the issue on a fresh Nova installation:

I haven't test this on mobile. I noticed the error on my own project, but tested it in the Nova Dusk Suite. Resizing the window to 1000px, the responsive menu is activated (great!), but when clicking on a link, the menu remains opened. The content changes, but the menu does not close.

I am not sure if this is related to the clicking and not tapping it, but I think the menu should close after clicking on a resource or othe link.

Thanks!

![image](https://user-images.githubusercontent.com/7245728/163756314-762d0a7b-dab9-4f35-af4f-5ec4ba1ca425.png)

**Comment by davidhemphill** [comment]
This has been fixed and will be included in the next release. 👍
